### DISCUSSION TITLE
Hide collection from collection types (local plugin)

### DISCUSSION THREAD

**Discussion by lodi-g** [strapi/strapi#8072] [Legacy Topics]
Hello,

I'm creating a plugin for Strapi, and it introduces new collections. I'd like to make the collection invisible from the 'collection types' panel (at the top of the left navbar).

Is this feature possible? 

Thanks

**Comment by lodi-g** [comment]
Nvm, it was answered by https://github.com/strapi/strapi/discussions/7809

**Reply by MattieBelt** [reply]
Hey @lodi-g, checkout this PR #6955.
